### ISSUE TITLE
Fine-tune on a different domain

### ISSUE THREAD

**Issue by tonybuih2** [IDEA-Research/Rex-Omni#40]
I tried your fine-tuning code on a different domain—specifically, the medical domain—which is quite different from your model’s original natural image domain. However, the performance is very poor. How can I improve it?

**Comment by PrAsAnNaRePo** [comment]
I fine-tuned Rex-Omni with sft with our custom data of engineering manuals. The final model does have the following problem:
- mis-classification
- lot of false positive

Training detils:
- Training sample size: 1.8k (5k annotated)
- Train loss: 1.33
- epoch: 2
- global step: 336

training command:
`/workspace/Rex-Omni/finetuning/train.py --config configs/sft.py --deepspeed scripts/zero2.json --data_flatten False --tune_mm_vision True --tune_mm_mlp True --tune_mm_llm True --bf16 --output_dir work_dirs/sft --num_train_epochs 2 --per_device_train_batch_size 4 --gradient_accumulation_steps 4 --eval_strategy no --save_strategy steps --save_steps 1000 --save_total_limit 5 --learning_rate 5e-6 --mm_projector_lr 2e-6 --vision_tower_lr 1e-6 --optim adamw_torch --warmup_ratio 0.03 --weight_decay 0.01 --max_grad_norm 1 --lr_scheduler_type cosine --logging_steps 1 --model_max_length 4096 --gradient_checkpointing True --dataloader_num_workers 4 --run_name rexomni_sft --report_to wandb`

I benchmarked the model in my custom dataset, it got scored only 0.07 mAP.
What did i miss here? How can i improve the model performance? Please help me guys!

**Comment by Mountchicken** [comment]
1. For such data, the model may not recognize these categories by itself, and you would need to perform SFT  first.

2. Just use the original resolution; our code will automatically resize it.
